regulations/templates/regulations/chrome.html

```diff
@@ -89,7 +89,7 @@
     {% block drawer-history %}
         <div id="timeline" class="history-drawer toc-container hidden">
             <div class="drawer-header">
-                <h3 class="toc-type">Regulation Timeline</h3>
+                <h2 class="toc-type">Regulation Timeline</h2>
```

Comment:
Looks like these types of changes would need to be added to `diff-chrome.html` as well?